features/developer_portal/admin/account/personal_details_sso.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Dev Portal Buyer Personal Details
+  As a buyer
+  I want to change my personal details
+
+  Background:
+    Given a buyer logged in to a provider using SSO
+
+  Scenario: Buyer shouldn't see any password input 
+    Given the buyer wants to edit their personal details
+    Then the buyer shouldn't see any reference to password
```

Comment:
This scenario is redundant, try to merge it with _the buyer edits their personal details_.